test/e2e/tests/test_scenarios.py

```diff
@@ -57,6 +121,11 @@ class UserHelper(helper.ResourceHelper):
     Overrides methods as required for custom resources.
     """
 
+    def get_aws_resource(self, resource_name):
+        users = self.mdb.describe_users(UserName=resource_name)
+        user = users['Users'][0]
+        return user
+
     def wait_for_delete(self, reference: k8s.CustomResourceReference):
         logging.debug(f"User - wait_for_delete()")
```

Comment:
Just curious here, how are these function waiting for the resources to be deleted? are they overridden somewhere?

Reply:
There is no operation for `wait_for_delete` in the controller. Does `runtime` call deletion again if the resource is not deleted? I remember I saw `reconciler` called `deleteResource ` again in the controller log after some default wait time?

Reply:
That's why I asked, what's the point of having a `wait_for_delete` is none of the resources will be hang a little bit before being deleted. But it's a nit please ignore.

Reply:
I can ask the original author about it.